code/datums/components/forensics.dm

```diff
@@ -0,0 +1,153 @@
+/datum/component/forensics
+	var/list/fingerprints		//assoc print = print
+	var/list/hiddenprints		//assoc ckey = realname/gloves/ckey
+	var/list/blood_DNA			//assoc dna = bloodtype
+	var/list/fibers				//assoc print = print
+
+/datum/component/forensics/InheritComponent(datum/component/forensics/F, original)
+	SANITIZE_LIST(fingerprints) |= F.fingerprints
+	SANITIZE_LIST(hiddenprints) |= F.hiddenprints
+	SANITIZE_LIST(blood_DNA) |= F.blood_DNA
+	SANITIZE_LIST(fibers) |= F.fibers

```

Comment:
I hate how you do this component and it has enough wrong with it that I don't want to untangle the chains of logic and over sanitation. Get this in a more readable state.

Also the usecase of `SANITIZE_LIST` where it isn't a list already will result in nothing happening because this will be `list() |+ F.fibers` aka no asignment